Chinese/crime_school/crime_school.md

```diff
@@ -0,0 +1,2 @@
+李杀神、王诛魔和刘斩仙三位少年做梦也没有想到会在狼牙区飞虎路私立断罪小学再度聚首，面对校园
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.